core/payment/src/wallet.rs

```diff
@@ -0,0 +1,40 @@
+use ya_core_model::driver::{driver_bus_id, Enter, Exit, Transfer};
+use ya_service_bus::typed as bus;
+
+pub async fn enter(
+    amount: String,
```

Comment:
Parsing amount should be done in CLI, not in driver.
```suggestion
    amount: BigDecimal,
```